content/fr/docs/reference/glossary/pod.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Pod
+id: pod
+date: 2018-04-12
+full_link: /docs/concepts/workloads/pods/pod-overview/
+short_description: >
+  Le plus petit et le plus simple des objets Kubernetes. Un Pod est un ensemble de conteneurs fonctionnant sur votre cluster.
+
+aka:
+tags:
+- core-object
+- fundamental
+---
+ Le plus petit et le plus simple des objets Kubernetes. Un Pod est un ensemble de {{< glossary_tooltip text="conteneurs" term_id="container" >}} fonctionnant sur votre cluster.
+
+<!--more-->
+
+Un Pod est généralement configuré pour faire fonctionner un seul conteneur primaire. Il peut également exécuter des conteneurs side-car optionnels qui ajoutent des fonctions supplémentaires comme le logging. Les Pods sont généralement gérés par un {{< glossary_tooltip="déploiement" term_id="deployment" >}}}.
```

Comment:
```suggestion
Un Pod est généralement configuré pour faire fonctionner un seul conteneur primaire. Il peut également exécuter des conteneurs side-car optionnels qui ajoutent des fonctions supplémentaires comme le logging. Les Pods sont généralement gérés par un {{< glossary_tooltip="déploiement" term_id="deployment" >}}.
```
I would be tempted to write _Deployment_ (with initial capitalization) rather than the localized term. The name Deployment is meaningful to the Kubernetes API server.

Reply:
Thank you for the suggestions! Both are included in the update.